ml/ContinuousGesturePredictor.py

```diff
@@ -151,6 +151,8 @@ def showStatistics(predictedClass, confidence):
         className = "Palm"
```

Comment:
Instead of adding if statements - let's change this to a dictionary with key - gesture name pairs